res/xml/method.xml

```diff
@@ -9,6 +9,7 @@
   <subtype android:label="%s" android:languageTag="es" android:imeSubtypeLocale="es_ES" android:imeSubtypeMode="keyboard" android:isAsciiCapable="true" android:imeSubtypeExtraValue="default_layout=qwerty_es,extra_keys=accent_grave|accent_aigu|accent_tilde|accent_trema|€"/>
   <subtype android:label="%s" android:languageTag="fr" android:imeSubtypeLocale="fr_FR" android:imeSubtypeMode="keyboard" android:isAsciiCapable="true" android:imeSubtypeExtraValue="default_layout=azerty,extra_keys=accent_grave|accent_aigu|accent_circonflexe|accent_cedille|accent_trema|€"/>
   <subtype android:label="%s" android:languageTag="hu" android:imeSubtypeLocale="hu_HU" android:imeSubtypeMode="keyboard" android:isAsciiCapable="true" android:imeSubtypeExtraValue="default_layout=qwertz_hu,extra_keys=accent_aigu|accent_trema|accent_ogonek|€"/>
+  <subtype android:label="%s" android:languageTag="he" android:imeSubtypeLocale="he_IL" android:imeSubtypeMode="keyboard" android:isAsciiCapable="true" android:imeSubtypeExtraValue="default_layout=he_il_1452_1"/>
```

Comment:
Here you can specify keys that will be added to every layouts with `,extra_keys=`. This can be useful for bidi markers if you are regularly switching between two layouts.

Reply:
Actually the bidi markers are always used when switching back and forth:
1. Write English; Switch to Hebrew; Write a word and finish with a question mark / exclamation mark; Insert RLM; Switch to English; Write English...
2. Write Hebrew; Switch to English; Write a word; Switch to Hebrew; Write question mark / exclamation mark; Insert LRM; Write Hebrew...

Reply:
That's what `extra_keys` is for.